test/unittest/unittest_nntrainer_tensor_neon_fp16.cpp

```diff
@@ -423,6 +423,62 @@ TEST(nntrainer_Tensor, dot_sgemm) {
   EXPECT_IN_RANGE((float)cosSimNeon, 0.99, 1);
 }
 
+TEST(nntrainer_Tensor, swish_i_p) {
+  int batch = 4;
+  int channel = 10;
+  int height = 10;
+  int width = 10;
+
+  const int MOD = 10;
+  const float eps = 1e-2;
+
+  nntrainer::Tensor input(batch, channel, height, width,
+                          nntrainer::Tformat::NCHW, nntrainer::Tdatatype::FP16);
+  GEN_TEST_INPUT(input, (i * (batch * height) + j * (width) + k) % MOD + 1);
+
+  nntrainer::Tensor ground_truth(batch, channel, height, width,
+                                 nntrainer::Tformat::NCHW,
+                                 nntrainer::Tdatatype::FP32);
+  GEN_TEST_INPUT(ground_truth,
+                 (i * (batch * height) + j * (width) + k) % MOD + 1);
+
+  input.swish_i();
+
+  for (int b = 0; b < batch; b++) {
+    for (int c = 0; c < channel; c++) {
+      for (int h = 0; h < height; h++) {
+        for (int w = 0; w < width; w++) {
+          ground_truth.setValue(
+            b, c, h, w,
+            ground_truth.getValue(b, c, h, w) /
+              (1 + std::exp(-1 * ground_truth.getValue(b, c, h, w))));
+        }
+      }
+    }
+  }
+
+  bool flag = true;
+
+  for (int b = 0; b < batch; b++) {
+    for (int c = 0; c < channel; c++) {
+      for (int h = 0; h < height; h++) {
+        for (int w = 0; w < width; w++) {
+          double err = std::abs(input.getValue<__fp16>(b, c, h, w) -
```

Comment:
We can compare the 2 tensor by using EXPECT_EQ.
`EXPECT_EQ(input, ground_truth`

And if possible please apply this code to recently added unittest like trigonometric_simd_cos